spec/lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation_spec.rb

```diff
@@ -474,6 +474,584 @@
+          expect(step_nav_links.show_related_links_for_secondary_step_by_steps?).to be false
+          expect(step_nav_links.show_header?).to be false
+          expect(step_nav_links.show_related_links?).to be false
+          expect(step_nav_links.show_sidebar?).to be false
+        end
+      end
+
+      context "for a content item with a couple `secondary_to_step_navs` links and a couple `related_to_step_navs` link" do
+        let(:content_store_response) do
+          {
+            "title" => "Book a session in the vomit comet",
+            "document_type" => "transaction",
+            "links" => {
+              "related_to_step_navs" => Array.new(2, related_to_step_nav),
+              "secondary_to_step_navs" => Array.new(2, step_nav),
+            }
+          }
+        end
+
+        it "parses the content item" do
+          step_nav_links = described_class.new(content_store_response, '/vomit-comet-session')
+
+          expect(step_nav_links.step_navs.count).to eq(0)
+          expect(step_nav_links.secondary_step_by_steps.count).to eq(2)
+          expect(step_nav_links.related_to_step_navs.count).to eq(2)
+
+          expect(step_nav_links.show_secondary_step_by_step?).to be false
+          expect(step_nav_links.show_related_links_for_secondary_step_by_steps?).to be false
+          expect(step_nav_links.show_header?).to be false
+          expect(step_nav_links.show_related_links?).to be false
+          expect(step_nav_links.show_sidebar?).to be false
+        end
+      end
+
+      context "for a content item with many `secondary_to_step_navs` links and many `related_to_step_navs` link" do
+        let(:content_store_response) do
+          {
+            "title" => "Book a session in the vomit comet",
+            "document_type" => "transaction",
+            "links" => {
+              "related_to_step_navs" => Array.new(6, related_to_step_nav),
+              "secondary_to_step_navs" => Array.new(6, step_nav),
+            }
+          }
+        end
+
+        it "parses the content item" do
+          step_nav_links = described_class.new(content_store_response, '/vomit-comet-session')
+
+          expect(step_nav_links.step_navs.count).to eq(0)
+          expect(step_nav_links.secondary_step_by_steps.count).to eq(6)
+          expect(step_nav_links.related_to_step_navs.count).to eq(6)
+
+          expect(step_nav_links.show_secondary_step_by_step?).to be false
+          expect(step_nav_links.show_related_links_for_secondary_step_by_steps?).to be false
+          expect(step_nav_links.show_header?).to be false
+          expect(step_nav_links.show_related_links?).to be false
+          expect(step_nav_links.show_sidebar?).to be false
+        end
+      end
+    end
+
+    context "secondary step by step with active step by step" do
+      it "shows header for related to step nav when a step by step is active and there is also a secondary step by step" do
```

Comment:
This can't be right, can it? I thought "related_to" step navs don't show on the content item?